front/app/containers/Authentication/steps/_components/NemLogInButton/index.tsx

```diff
@@ -20,7 +20,7 @@ interface Props {
   grayBorder?: boolean;
   standardSSOBehavior?: boolean;
   onClick?: (method: TVerificationMethod) => void;
-  onClick2?: () => void;
+  onClickStandardSSO?: () => void;
```

Comment:
This component is a total mess, I just hacked it together at 2am as fast as possible last time. Made a note to improve it